src/components/Button/Button.tsx

```diff
@@ -19,44 +15,14 @@ export interface Props {
    * Disables the field and prevents editing the contents
    */
   disabled?: boolean;
-  /**
-   * Toggles button that fills the full width of its container
-   */
-  fullWidth?: boolean;
-  /**
-   * Visually hide button text (but text is still accessible to assistive technology)
-   */
-  hideText?: boolean;
```

Comment:
I'm guessing this is for the case where you have just an icon with some screen reader text. The way we're doing this now is just passing in an icon and putting the screen reader text on the icon. I think that API is a little more obvious, but it hinges on people passing in their own icons. Another option is to just have people pass in screen reader-only text (in `traject` they use just put an `sr-only` class on the element) rather than passing in `text` and separately also passing in `hideText`.

Reply:
I buy it!

Reply:
Remove this and added `aria-label` instead 👍